CMakeLists.txt

```diff
@@ -6,6 +6,11 @@
 #   schmieder.matthias@gmail.com
 
 cmake_minimum_required(VERSION 3.13)
+project(minuzip-ng
```

Comment:
GitHub diff does not show where original `project` call is removed? 
Also project name is mis-spelled.
And original `project` call looks like:
```cmake
project(minizip${MZ_PROJECT_SUFFIX} LANGUAGES C VERSION ${VERSION})

```

Reply:
Fixed that typo (sorry).
I'm not sure is it will be working with variable in project name.
If you know that it will work just merge that PR and continue 😄

Reply:
With multiline project() I think that it will be a bit easier to update VERSION.
I've addded as well project descriotion which could make project() line a bit to long if it will be in single line.

Reply:
> I'm not sure is it will be working with variable in project name.

Why not?

> With multiline project()

I don't mind multiline.

Reply:
I don't know how such modyfication test a bit .. if you know that just please continue adding project() with variable.

Reply:
Why change `project()` all?